packages/venia-ui/lib/components/OrderHistoryPage/orderRow.js

```diff
@@ -39,12 +40,15 @@ const OrderRow = props => {
     const hasShipment = !!shipments.length;
     const derivedStatus =
```

Comment:
This line is super hard to read, more so now with the `formatMessage` functions. I wonder if we can refactor this to  make my brain hurt less, not a requirement for approval though.

Reply:
LOL yea it is pretty rough.